playbook-website/app/views/samples/crm_client_list/index.html.erb

```diff
@@ -59,7 +59,11 @@
 
   <%= pb_rails("flex", props: {classname: "flex-container", spacing: "between"}) do %>
     <%= pb_rails("flex/flex_item") do %>
-        <%= action.submit props: { text: "Apply", data: { disable_with: "<i class='far fa-spinner fa-spin mr-3'></i>Searching...".html_safe },}%>
+        <%= action.submit props: {
+        text: "Apply",
+        data: {
+          disable_with: "<i>pb_rails('icon', props: { icon: 'spinner', spin: true, fixed_width: true, marginRight: "xs"})</i>Searching...".html_safe
```

Comment:
@skduncan , I don't think we should be adding marginRight as with 'mr-3', there were no margins because the class doesn't exist.